.github/workflows/release.yml

```diff
@@ -32,7 +32,7 @@ jobs:
       uses: goreleaser/goreleaser-action@v6
       with:
         version: latest
-        args: release --rm-dist
+        args: release --clean
```

Comment:
This was due to the goreleaser/goreleaser-action action update. I didn't tested that the release actually works at that time.